server/src/main/java/org/elasticsearch/action/get/TransportShardMultiGetAction.java

```diff
@@ -155,4 +150,21 @@ protected String getExecutor(MultiGetShardRequest request, ShardId shardId) {
             return super.getExecutor(request, shardId);
         }
     }
+
+    private void asyncShardMultiGet(MultiGetShardRequest request, ShardId shardId, ActionListener<MultiGetShardResponse> listener)
```

Comment:
```suggestion
    private void asyncMultiGet(MultiGetShardRequest request, ShardId shardId, ActionListener<MultiGetShardResponse> listener)
```

Just to mirror the `asyncGet`

Reply:
My intention was to use the same name as the action itself, hence the "shard" in the name. `TransportMultiGetAction` is slightly different and also exists (and gets broken down into `TransportShardMultiGetAction`).

Reply:
Ok